packages/web/src/components/App.tsx

```diff
@@ -18,10 +19,25 @@ export function App({ locationChangeHandler }: AppProps) {
   const location = useLocation();
   locationChangeHandler(location.pathname);
   return (
-    <Routes>
-      <Route index element={<HomePage />} />
-      <Route path='/w/:hostname' element={<WebsiteResultsPage />} />
-      <Route path='*' element={<Navigate replace to='/' />} />
-    </Routes>
+    <>
+      <Helmet>
+        <title>Production Webpack Bundle Analyzer - GradeJS</title>
+        <meta
+          name='description'
+          content='GradeJS analyzes production JavaScript files and matches bundled NPM packages with specific version precision.'
+        />
+        <meta property='og:title' content='Production Webpack Bundle Analyzer - GradeJS' />
```

Comment:
Seems like `og:url` is also required. Can you please add it?
<img width="815" alt="image" src="https://user-images.githubusercontent.com/5869473/189068253-3ac8b31e-f04a-4438-a69a-7a01341b9943.png">